auslib/web/admin/views/rules.py

```diff
@@ -23,10 +23,11 @@ def process_rule_form(form_data):
 
     mapping_choices = [(item['name'], item['name']) for item in release_names]
 
-    if 'locale' in form_data:
-        locale = form_data['locale']
-        if locale is not None:
-            form_data['locale'] = ''.join(locale.split())
+    csv_columns = ['locale', 'distribution']
+    for column in csv_columns:
+        column_data = form_data.get(column)
+        if column_data:
+            form_data[column] = ''.join(column_data.split())
```

Comment:
@mozbhearsum, let me know if you think that we should remove duplicates to save space.